src/popover/types.js

```diff
@@ -83,6 +83,8 @@ export type BasePopoverPropsT = {
   onClick?: (e: Event) => mixed,
   /** Handler for 'Esc' keypress events */
   onFocus?: (e: Event) => mixed,
+  /** Pass FocusOptions to autoFocus */
+  autoFocusOptions?: FocusOptions,
```

Comment:
I think autoFocusOptions makes more sense imo, because this focusOptions is set & applicable for only autoFocus, but I can change that to focusOptions too to mimic the prop of react-focus-lock